pkgs/desktops/plasma-5/kde-gtk-config/default.nix

```diff
@@ -13,7 +13,6 @@ mkDerivation {
     ki18n kio glib gtk2 gtk3 karchive kcmutils kconfigwidgets kiconthemes
     knewstuff gsettings-desktop-schemas
   ];
-  patches = [ ./patches/follow-symlinks.patch ./patches/gsettings.patch ];
```

Comment:
Have we confirmed that these packages are actually not necessary? I recall, in particular, that `kde-gtk-config` was completely broken without `follow-symlinks.patch`.

Reply:
Please refer to this comment on the state of gtk-config:

> I could use some help with the gtk config page.
> The plasma kde-gtk-config module was mostly rewritten from scratch. The menu was moved to the other Application Style code, and kde-gtk-config now only contains a kded plugin. If the settings menu detects that the kded kde-gtk-config module is loaded, it will show the settings page.
> I don't know how to figure out if the kded module has been loaded or where exactly the problem is, but the settings page is not shown.

So no, without patches it doesn't work, but since source was 100% replaced with new code upstream, the patches won't apply and can not be easily rewritten.

Reply:
The answer to your question is: No, we haven't confirmed. Quite the opposite, we have confirmed that it doesn't work without patches, which is also the reason why figuring out the new way to do this is part of the todo list in the initial comment.

Reply:
As I recall, the patches mainly affected the code for the settings menu. The code that needs to be patched has probably moved to `systemsettings`. I guess we may _also_ need to patch the `kded` module, but I can't imagine what that module is actually doing: setting the GTK theme is a matter of writing a few static files, so I don't know why we need a running daemon to do that.

Reply:
Some more information on this new "Plasma GTKd" is available here:
https://phabricator.kde.org/D24743

Reply:
And the new settings page is added here: https://phabricator.kde.org/D26583

Reply:
I think the new code already follows symlinks, at least the only place where I can find something that looks similar to the code that was previously patched (plasma-desktop/kcms/style/gtkthemesmodel.cpp L. 154) uses QDir().entryList without passing QDir::NoSymlinks.